ext/integrations/integrations.c

```diff
@@ -176,7 +176,25 @@ void ddtrace_integrations_minit(void) {
     DD_SET_UP_DEFERRED_LOADING_BY_METHOD(DDTRACE_INTEGRATION_LAMINAS, "Laminas\\Mvc\\Application", "init",
                                          "DDTrace\\Integrations\\Laminas\\LaminasIntegration");
     DD_SET_UP_DEFERRED_LOADING_BY_METHOD(DDTRACE_INTEGRATION_LAMINAS, "Laminas\\Mvc\\Application", "__construct",
-                                             "DDTrace\\Integrations\\Laminas\\LaminasIntegration");
+                                         "DDTrace\\Integrations\\Laminas\\LaminasIntegration");
+    DD_SET_UP_DEFERRED_LOADING_BY_METHOD(DDTRACE_INTEGRATION_LAMINAS, "Laminas\\Log\\LoggerInterface", "emerg",
+                                         "DDTrace\\Integrations\\Laminas\\LaminasIntegration");
+    DD_SET_UP_DEFERRED_LOADING_BY_METHOD(DDTRACE_INTEGRATION_LAMINAS, "Laminas\\Log\\LoggerInterface", "alert",
+                                         "DDTrace\\Integrations\\Laminas\\LaminasIntegration");
+    DD_SET_UP_DEFERRED_LOADING_BY_METHOD(DDTRACE_INTEGRATION_LAMINAS, "Laminas\\Log\\LoggerInterface", "crit",
+                                         "DDTrace\\Integrations\\Laminas\\LaminasIntegration");
+    DD_SET_UP_DEFERRED_LOADING_BY_METHOD(DDTRACE_INTEGRATION_LAMINAS, "Laminas\\Log\\LoggerInterface", "err",
+                                         "DDTrace\\Integrations\\Laminas\\LaminasIntegration");
+    DD_SET_UP_DEFERRED_LOADING_BY_METHOD(DDTRACE_INTEGRATION_LAMINAS, "Laminas\\Log\\LoggerInterface", "warn",
+                                         "DDTrace\\Integrations\\Laminas\\LaminasIntegration");
+    DD_SET_UP_DEFERRED_LOADING_BY_METHOD(DDTRACE_INTEGRATION_LAMINAS, "Laminas\\Log\\LoggerInterface", "notice",
+                                         "DDTrace\\Integrations\\Laminas\\LaminasIntegration");
+    DD_SET_UP_DEFERRED_LOADING_BY_METHOD(DDTRACE_INTEGRATION_LAMINAS, "Laminas\\Log\\LoggerInterface", "info",
+                                         "DDTrace\\Integrations\\Laminas\\LaminasIntegration");
+    DD_SET_UP_DEFERRED_LOADING_BY_METHOD(DDTRACE_INTEGRATION_LAMINAS, "Laminas\\Log\\LoggerInterface", "debug",
+                                         "DDTrace\\Integrations\\Laminas\\LaminasIntegration");
+    DD_SET_UP_DEFERRED_LOADING_BY_METHOD(DDTRACE_INTEGRATION_LAMINAS, "Laminas\\Log\\LoggerInterface", "log",
+                                         "DDTrace\\Integrations\\Laminas\\LaminasIntegration");
```

Comment:
Do we actually need these, or is that covered by the Laminas\Mvc\Application hook?

Reply:
I think it all depends on if we go from the fact that if someone uses laminas-log, he also uses the laminas framework. For instance, this is not the case in the tests where I'm only using the library.

Reply:
I removed these and loaded the integration explicitly in the test file instead.